examples/todolist/.env.example

```diff
@@ -0,0 +1,3 @@
+REACT_APP_BACKEND=http://localhost:3001
+REACT_APP_RUM_BEACON_URL=https://rum-ingest.<REALM>.signalfx.com/v1/rum
```

Comment:
I think that we should default to US0 here for ease of use, maybe add a comment that it might need to be changed.